kornia/morphology/top_hat.py

```diff
@@ -0,0 +1,27 @@
+from kornia.morphology import open
+import torch
+
+
+def top_hat(tensor: torch.Tensor, kernel: torch.Tensor) -> torch.Tensor:
```

Comment:
Logistic question: why do you prefer creating a separate file for each operation?

Reply:
No particular reason, i just created one after another.
I ordered the scripts in similar functions (dilate/erode) (open/close) (others). 
Could be better in that way?